gulpfile.js

```diff
@@ -1,70 +1,43 @@
-const gulp = require('gulp');
+const { dest, series, src, watch } = require('gulp');
 const gutil = require('gulp-util');
 const cp = require('child_process');
 const sass = require('gulp-sass');
 const rollup = require('rollup');
 const babel = require('rollup-plugin-babel');
-const externalHelpers = require('babel-plugin-external-helpers');
 const browserSync = require('browser-sync').create();
 const header = require('gulp-header');
-const runSequence = require('run-sequence');
 const uglify = require('gulp-uglify');
-const pump = require('pump');
 const rename = require('gulp-rename');
 const postcss = require('gulp-postcss');
 const cssnano = require('gulp-cssnano');
-const autoprefixer = require('autoprefixer')
+const autoprefixer = require('autoprefixer');
 const package = require('./package.json');
 
 // Get the current year for copyright in the banner
 const currentYear = new Date().getFullYear();
```

Comment:
Forgot to mention this before. Let's replace this dynamically-created date with a hard-coded `2019` since once something is open sourced the copy right date should only ever be the year it was first released.